### PR TITLE
Complete the hw 3.

### PR DESCRIPTION
**Pull request by AlinaZvarych**

### CODE REVIEW

File: src/main/java/hw_3/Task1.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import java.util.Arrays;
+
+public class Task1 {
+    public static void main(String[] args) {
+        double[] numbers = {3.43, 12.1212, 12.22, 2.29};
+
+        String result = Arrays.toString(numbers)
+                              .replace(", ", ";");
+
+        System.out.println(result);
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: Realization is incorrect. You need to use cycle here to work with each element from numbers array. And combine each element with string and ; symbol

File: src/main/java/hw_3/Task2.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package hw_3;
+
+public class Task2 {
+    public static void main(String[] args) {
+
+        String sentence = "Mother washed hands.";
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task2.java:5]: Please remove extra line

File: src/main/java/hw_3/Task2.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        String[] wordsArray = sentence.split(" ");
+        for (String word : wordsArray) {
+
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task2.java]: Please remove extra line

File: src/main/java/hw_3/Task3.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package hw_3;
+
+public class Task3 {
+    public static void main(String[] args) {
+
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task3.java]: Please remove extra line

File: src/main/java/hw_3/Task3.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+        String sentence = "Mother washed hands.";
+        String modifiedSentence = sentence.replace(" ", "\n");
+
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task3.java]: Please remove extra line

File: src/main/java/hw_3/Task6.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
+package hw_3;
+
+public class Task6 {
+        public static void main(String[] args) {
+            int i = 124;
+            byte b = 66;
+            double d = 77.098;
+
+            Number[] array = new Number[3];
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task6.java]: array type is incorrect.
Screenshot_13

File: src/main/java/hw_3/Task7.java [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+                    {1, 2, 3},
+                    {4, 5, 6},
+                    {7, 8, 9}
+            };
+            for (int i = 0; i < numbers.length; i++) {
+                for (int j = 0; j < numbers[i].length; j++) {
+
+                    System.out.print(numbers[i][j] + " ");
+                }
+                System.out.println();
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task7.java]: THis System.out.println can be removed? If you need to type something from the new line you can use '\n'

File: src/main/java/hw_3/Task8.java [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+            char[] charArray = sentence.substring(6, 11).toCharArray();
+
+            System.out.print("Array of symbols: ");
+
+            for (char c : charArray) {
+                System.out.print(c + " ");
+            }
+
+            System.out.println();
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task8.java]: Can be removed? If you need to type something from the new line you can use '\n'

File: src/main/java/hw_3/Task1.java (hunk 1 above)
Comment (AlinaZvarych) [reply src/main/java/hw_3/Task1.java]: Done

File: src/main/java/hw_3/Task2.java (hunk 2 above)
Comment (AlinaZvarych) [reply src/main/java/hw_3/Task2.java:5]: Done

File: src/main/java/hw_3/Task2.java (hunk 3 above)
Comment (AlinaZvarych) [reply src/main/java/hw_3/Task2.java]: Done

File: src/main/java/hw_3/Task3.java (hunk 4 above)
Comment (AlinaZvarych) [reply src/main/java/hw_3/Task3.java]: Done

File: src/main/java/hw_3/Task3.java (hunk 5 above)
Comment (AlinaZvarych) [reply src/main/java/hw_3/Task3.java]: Done

File: src/main/java/hw_3/Task6.java (hunk 6 above)
Comment (AlinaZvarych) [reply src/main/java/hw_3/Task6.java]: Done

File: src/main/java/hw_3/Task7.java (hunk 7 above)
Comment (AlinaZvarych) [reply src/main/java/hw_3/Task7.java]: Done

File: src/main/java/hw_3/Task8.java (hunk 8 above)
Comment (AlinaZvarych) [reply src/main/java/hw_3/Task8.java]: Done (but i'm not sure if this is correct (and previous one as well))

**Pull request merged**